GenderPayGap.WebUI/Views/Compare/ComparisonBasket.cshtml

```diff
@@ -11,7 +11,7 @@
 }
 
 <div id="comparison-basket"
-     class="no-print show-empty js-stick-at-top-when-scrolling @(compareCount < 1 ? "visuallyhidden" : "")"
+     class="comparison-basket no-print show-empty js-stick-at-top-when-scrolling @(compareCount < 1 ? "visuallyhidden" : "")"

```

Comment:
Do we still need ` @(compareCount < 1 ? "visuallyhidden" : "")`?

Reply:
Yes, this hides the green basket banner on the search/employer pages when there is nothing in the basket. This functionality needs to stay the same until(if) we change from the green banner basket on those pages